notebooks/234-blip-visual-language-processing/234-blip-visual-language-processing.ipynb

```diff
@@ -0,0 +1,766 @@
+{
```

Comment:
<p>Could you please include some link to some terms, such as: tokenized question, output image embeddings, dynamic shapes, dynamic axes. Not all devs are familiar with those terms.</p><p> </p><p>It would be useful to add a flowchart here to explain what is Text encoder.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>too much flowcharts for one notebook I think... I maybe add it later, but do  not think that text encoder is something special which required own flowchart . Some developers do not know what is Deep Learning at all, it does not mean that we need to explain how convolution work in our notebooks and also should not teach how to train model (excluding optimization cases), because it is not our product targets. </p><p>Let's do not turn tutorial for demonstration openvino capabilities to expressive guide how VLP models works where flowcharts more then real code...</p>

Reply:
maybe as follow up in next request